hypha/apply/funds/models/submissions.py

```diff
@@ -1008,7 +1008,7 @@ def get_or_create_for_user(self, submission, reviewer):
             else:
                 groups = {REVIEWER_GROUP_NAME}
         elif not groups:
-            if reviewer.is_staff or reviewer.is_superuser:
```

Comment:
I think this is a case where is_staff etc. really should be used.

Reply:
I think we are checking the same thing in `is_apply_staff`.  `STAFF_GROUP_NAME` or `is_superuser`.
Or it might be any other custom `is_staff`?
okay, let me check.

Reply:
Yes wherever we are setting is_staff=True, we are adding STAFF_GROUP_NAME in user groups. 
And in property, is_apply_staff we are checking for both of the things if the user has STAFF_GROUP_NAME or is_superuser. 
I am not getting any edge cases, can you please enlighten it for me.

Reply:
@frjo I got your point now. I changed it accordingly.